UI/win-update/updater/updater.cpp

```diff
@@ -531,7 +527,12 @@ static inline DWORD WaitIfOBS(DWORD id, const wchar_t *expected)
 	if (!proc.Valid())
 		return WAITIFOBS_WRONG_PROCESS;
 
-	if (!GetProcessImageFileName(proc, path, _countof(path)))
+	if (!GetModuleFileNameExW(proc, nullptr, path, _countof(path)))
```

Comment:
Perhaps `QueryFullProcessImageNameW` is better here?

Reply:
Either seems to work. Given the note in the MS Docs about `GetModuleFileName` being primarily intended for use in debuggers I've updated it to use `QueryFullProcessImageName` instead.